timelock-agent/src/main/java/com/palantir/timelock/config/TimeLockInstallConfiguration.java

```diff
@@ -23,6 +23,11 @@
     ClusterConfiguration cluster();
 
     @Value.Default
+    default TsBoundPersisterConfiguration timestampBoundPersistence() {
```

Comment:
Not comfortable with a default value here. Let's just make it required.

Reply:
Actually, I'm okay with this having a default value of "use paxos" as long as we clean up the `PaxosTsBoundPersisterConfiguration` per my comments on that class.

Reply:
Done.